packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1079,6 +1084,13 @@ class PopupMenuButton<T> extends StatefulWidget {
   /// Theme.of(context).cardColor is used.
   final Color? color;
 
+  /// If provided, menu padding is used for the popup menu.
```

Comment:
Can you make it clear whether this is padding around the individual items in the popup menu, or around the outside of the popup menu?

Reply:
Resolved in the latest commit